components/publish/PublishAttachment.vue

```diff
@@ -51,26 +51,60 @@ function toggleApply() {
       py-6
       px-6 max-w-300
     >
+      <div flex flex-justify-end mb>
+        <button btn-text p-0 @click="isEditDialogOpen = false">
```

Comment:
this works, but I'm still so confused about the `= false`

Reply:
That closes the modal